code/game/objects/items/crayons.dm

```diff
@@ -44,20 +101,28 @@
 	update_window(user)
 
 /obj/item/toy/crayon/mime/update_window(mob/living/user as mob)
-	dat += "<center><span style='border:1px solid #161616; background-color: [colour];'>&nbsp;&nbsp;&nbsp;</span><a href='?src=\ref[src];color=1'>Change color</a></center>"
+	dat += includeColor()
 	..()
 
 /obj/item/toy/crayon/mime/Topic(href,href_list)
-	if ((usr.restrained() || usr.stat || usr.get_active_hand() != src))
-		return
-	if(href_list["color"])
-		if(colour != "#FFFFFF")
-			colour = "#FFFFFF"
-		else
-			colour = "#000000"
-		update_window(usr)
-	else
-		..()
+	topicColor(href,href_list,TRUE)
+	..()
+
+/obj/item/toy/crayon/spraycan/mime
+	icon_state = "spraycanmime_cap"
+	desc = "A very sad-looking spraycan."
+	colour = "#FFFFFF"
+	colourName = "mime"
+	uses = 0
+
+/obj/item/toy/crayon/spraycan/mime/update_window(mob/living/user as mob)
+	dat += includeColor()
+	..()
+
+/obj/item/toy/crayon/spraycan/mime/Topic(href,href_list)
+	topicColor(href,href_list,TRUE)
+	..()
+

```

Comment:
I made them into a set of procs because of spam-pasting. Without the includeColor() and topicColor() proc, there would be a copy-paste of crayon code here.